lib/changelog.rb

```diff
@@ -0,0 +1,111 @@
+# A class for representing a changelog consisting of several changes
+# over a certain timespan (between two commits)
+class Changelog
+  def initialize(changes, tag_from = nil, tag_to = nil, from_commit = nil, to_commit = nil)
+    @fixes = changes.select { |c| c.type == Change::FIX }
+    @features = changes.select { |c| c.type == Change::FEAT }
+    @gui_changes = changes.select { |c| c.type == Change::GUI }
+    @refactorings = changes.select { |c| c.type == Change::REFACTORING }
+    @tag_from = tag_from
+    @tag_to = tag_to
+    @commit_from = from_commit
+    @commit_to = to_commit
+  end
+
+  def changes
+    {
+      fixes: @fixes.map(&:note),
+      features: @features.map(&:note),
+      gui: @gui_changes.map(&:note),
```

Comment:
sneaky breaking change for my fastlane integration :laughing: 
But I like it, its consistent with the tokens.

However, it should be noted as a `refactoring` change!